helper.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
+
+const Variables = require('sq-toolkit/variables');
+const Exception = require('sq-toolkit/exception');
+const BigQuery = require('@google-cloud/bigquery').BigQuery;
+
+let _instance = null;
+
+const ErrorCode = {
+    ERROR_INSTANCE_ALREADY_CREATED: 'ERROR_INSTANCE_ALREADY_CREATED',
+    ERROR_INSTANCE_NOT_CREATED: 'ERROR_INSTANCE_NOT_CREATED'
+};
+
+class BigQueryHelper {
```

Comment:
why you are implementing this as singleton if you only use static methods?

Reply:
The idea is to initialize BigQuery in just ONE place in the apps we use this library. This way, when we instance a BigQueryJob, we will get the instance we created at that specific place. If you want to create an instance besides the singleton, you can do it with `create()` and then pass it to the `BigQueryJob` constructor